commonutils/src/main/java/io/intrepid/commonutils/AndroidUtils.java

```diff
@@ -8,6 +8,7 @@
 import android.view.View;
 import android.view.inputmethod.InputMethodManager;
 
+@SuppressWarnings({"WeakerAccess", "unused"})
```

Comment:
I don't think we should suppress the unsued warning since that's also an indicator that we should write a test for it